src/DssSpell.sol

```diff
@@ -52,22 +52,159 @@ contract DssSpellAction is DssAction {
+            flip: UNIV2WBTCDAI_FLIP,
+            pip: UNIV2WBTCDAI_PIP,
+            isLiquidatable: true,
+            isOSM: true,
+            whitelistOSM: false,
+            ilkDebtCeiling: 3 * MILLION,
+            minVaultAmount: 2000,
+            maxLiquidationAmount: 50000,
+            liquidationPenalty: 1300,
+            ilkStabilityFee: THREE_PCT,
+            bidIncrease: 300,
+            bidDuration: 6 hours,
+            auctionDuration: 6 hours,
+            liquidationRatio: 12500
+        });
+        addNewCollateral(UNIV2WBTCDAI_A);
+        DssExecLib.setChangelogAddress("UNIV2WBTCDAI",             UNIV2WBTCDAI_GEM);
+        DssExecLib.setChangelogAddress("MCD_JOIN_UNIV2WBTCDAI_A",  UNIV2WBTCDAI_JOIN);
+        DssExecLib.setChangelogAddress("MCD_FLIP_UNIV2WBTCDAI_A",  UNIV2WBTCDAI_FLIP);
+        DssExecLib.setChangelogAddress("PIP_UNIV2WBTCDAI",         UNIV2WBTCDAI_PIP);
+
+        // Onboard UNIV2AAVEETH-A
+        DssExecLib.addReaderToMedianWhitelist(
+            LPOracle(UNIV2AAVEETH_PIP).orb0(),
+            UNIV2AAVEETH_PIP
+        );
+        DssExecLib.addReaderToMedianWhitelist(
+            LPOracle(UNIV2AAVEETH_PIP).orb1(),
+            UNIV2AAVEETH_PIP
+        );
+        CollateralOpts memory UNIV2AAVEETH_A = CollateralOpts({
+            ilk: UNIV2AAVEETH_ILK,
+            gem: UNIV2AAVEETH_GEM,
+            join: UNIV2AAVEETH_JOIN,
+            flip: UNIV2AAVEETH_FLIP,
+            pip: UNIV2AAVEETH_PIP,
+            isLiquidatable: true,
+            isOSM: true,
+            whitelistOSM: false,
+            ilkDebtCeiling: 3 * MILLION,
+            minVaultAmount: 2000,
+            maxLiquidationAmount: 50000,
+            liquidationPenalty: 1300,
+            ilkStabilityFee: FOUR_PCT,
+            bidIncrease: 300,
+            bidDuration: 6 hours,
+            auctionDuration: 6 hours,
+            liquidationRatio: 16500
+        });
+        addNewCollateral(UNIV2AAVEETH_A);
+        DssExecLib.setChangelogAddress("UNIV2AAVEETH",             UNIV2AAVEETH_GEM);
+        DssExecLib.setChangelogAddress("MCD_JOIN_UNIV2AAVEETH_A",  UNIV2AAVEETH_JOIN);
+        DssExecLib.setChangelogAddress("MCD_FLIP_UNIV2AAVEETH_A",  UNIV2AAVEETH_FLIP);
+        DssExecLib.setChangelogAddress("PIP_UNIV2AAVEETH",         UNIV2AAVEETH_PIP);
+
+        // Dai Savings Rate Adjustment
+        PotAbstract(DssExecLib.pot()).drip();
+        DssExecLib.setDSR(ONE_HUNDREDTH_PCT);
+
+        // Remove Permissions for Liquidations Circuit Breaker
+        address flipperMom = DssExecLib.flipperMom();
+        DssExecLib.deauthorize(DssExecLib.flip(PSM_USDC_A_ILK), flipperMom);
+        DssExecLib.deauthorize(DssExecLib.flip(UNIV2DAIUSDC_A_ILK), flipperMom);
+
+        // Fix for Line != sum lines rounding error issue (0.602857457497899800874246318932698818152722680 DAI)
+        VatAbstract vat = VatAbstract(DssExecLib.vat());
+        vat.file("Line", add(vat.Line(), 602857457497899800874246318932698818152722680));
```

Comment:
We might want to avoid the use of safe math here in favor of a clearer `+`. As in this case the numbers are quite controlled. For example below we are using `now + 30 days`.
In this case I think the shorter we can do the code and the less back and forth we have on reviewing is the better.